commands/k8s_cluster.go

```diff
@@ -212,6 +212,7 @@ func RunK8sClusterGet(c *core.CommandConfig) error {
 	if err := utils.WaitForState(c, GetStateK8sCluster, viper.GetString(core.GetFlagName(c.NS, config.ArgK8sClusterId))); err != nil {
 		return err
 	}
+	c.Printer.Infof("K8s with id: %v is getting...", viper.GetString(core.GetFlagName(c.NS, config.ArgK8sClusterId)))
```

Comment:
Whould it make more sense to update to K8s Cluster with id...? Since there are multiple resources under `ionosctl k8s`  command.

Reply:
done

Reply:
yes